Classes/Updates/FileLocationUpdater.php

```diff
@@ -19,6 +19,7 @@
 use TYPO3\CMS\Core\Core\Environment;
 use TYPO3\CMS\Core\Database\ConnectionPool;
 use TYPO3\CMS\Core\Database\Query\Expression\ExpressionBuilder;
+use TYPO3\CMS\Core\Log\Logger;
```

Comment:
You added this namespace, but removed the `$logger` property, so this is never used. This seems odd.
```suggestion
```